entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -52,8 +48,9 @@ class EntityService(
         }
 
         ngsiLdEntity.geoProperties.forEach { ngsiLdGeoProperty ->
-            // TODO we currently don't support multi-attributes for geoproperties
-            createLocationProperty(ngsiLdEntity.id, ngsiLdGeoProperty.name, ngsiLdGeoProperty.instances[0])
+            ngsiLdGeoProperty.instances.forEach { ngsiLdGeoPropertyInstance ->
+                createGeoProperty(ngsiLdEntity.id, ngsiLdGeoProperty.name, ngsiLdGeoPropertyInstance)
+            }
```

Comment:
hmm, not exactly.

with this PR, we now support `location`, `operationSpace` and `observationSpace`, but I don't think a multi-instance version of `location` will work (and if not, it's not planned to be done in the scope of this PR).

Reply:
Ok we can only have one of the three geoproperties per entity ?

Reply:
Ok I changed this